src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -134,26 +146,34 @@ func importConfig(
 		// Special case 400 (validation) errors with friendly output
 		// about which configuration properties were invalid.
 		if he.Response.StatusCode == 400 {
-			fmt.Fprint(os.Stderr, formatValidationError(err, he))
+			ve, err := formatValidationError(err, he)
+			if err != nil {
+				return fmt.Errorf("error setting config: %v", err)
+			}
+			fmt.Fprint(os.Stderr, ve)
 			out.Die("No changes were made.")
 		}
 	}
 
 	// If we didn't handle a structured 400 error, check for other errors.
-	out.MaybeDie(err, "error setting config: %v", err)
+	if err != nil {
+		return fmt.Errorf("error setting config: %v", err)
```

Comment:
Why this change here? The above can `out.Die`?

Reply:
Yes, I tried to remove every os exit (either MaybeDie or Die) from the importConfig function.